Cargo.toml

```diff
@@ -62,13 +62,14 @@ tracing-test = { version = "0.2.4", features = ["no-env-filter"] }
 duplicate = "1.0.0"
 aho-corasick = "0.7.20"
 regex = "1.8.4"
-srclib = { git = "https://github.com/fossas/foundation-libs", branch = "add-srclib" }
+srclib = { git = "https://github.com/fossas/foundation-libs" }
 walkdir = "2.3.3"
 tar = "0.4.38"
 libflate = "1.3.0"
 typed-builder = "0.14.0"
 which = "4.4.0"
 tikv-jemallocator = { version = "0.5.0", optional = true }
+deadqueue = "0.2.4"
```

Comment:
According to the deadqueue docs, we need to add the `limited` feature if we want to use it: https://docs.rs/deadqueue/latest/deadqueue/.

I'm surprised that it works without it 🤔

Reply:
It is actually there by default! https://docs.rs/crate/deadqueue/latest/features